code/game/objects/items/crayons.dm

```diff
@@ -488,7 +488,7 @@
 
 	charges = -1
 
-/obj/item/toy/crayon/rainbow/afterattack(atom/target, mob/user, proximity)
+/obj/item/toy/crayon/rainbow/afterattack(atom/target, mob/user, proximity, params)
```

Comment:
what does adding that parameter do?

Reply:
https://github.com/tgstation/tgstation/blob/227983a777c0c76be3a96266bf5679aa6b12524e/code/game/objects/items/crayons.dm#L343-L349

passes the click location

Reply:
oh, neat. ty